docs/config/cluster.rst

```diff
@@ -709,31 +709,35 @@ to forced awareness or allocation filtering.
 Cluster-wide allocation filtering
 .................................
 
-Allow to control the allocation of all shards based on include/exclude filters.
+Allows to control whether shards can be allocated onto a node. If such a filter
+prevents shards from being allocated to a node this means for one thing, that
+new shards won't be allocated onto those nodes and at the same time that
+existing shards, which were on the node before, will be moved away, since they
+no longer fit the node based update criteria.
```

Comment:
```suggestion
Control which shards are allocated to which nodes.

Filter definitions are retroactively enforced. If a filter prevents matching shards from being newly allocated to a node, existing matching shards will also be moved away.
```

Reply:
this suggestion has broken github. but basically, I am proposing you replace your para with these two paras completely